adr/0026-schema.md

```diff
@@ -106,6 +132,7 @@ build: # Everything here is created by Zarf not be users
     - missing-component
   flavor: cool-flavor
   lastNonBreakingVersion: "v0.99.9"
+  aggregateChecksum: shasum # this is moved from .metadata
```

Comment:
still cool with this being deprecated if it doesnt effect anything

Reply:
I believe we want to keep it, just move it off metadata. Basically Zarf creates a checksum.txt file with all the sha's of every file that compromises a Zarf package. We then take the sha of the contents of that checksum.txt and that becomes the aggregate checksum. It allows us to ensure at deploy time the package is exactly what we wanted it to be immediately after create.

Reply:
Its supposed to be on `build`, its a bit of an oopsie that its on a user-facing configuration

Reply:
It actually _cannot_ be removed as it is used for package integrity

signature --> zarf.yaml --> aggregateChecksum --> checksums.txt